### PR TITLE
scaling FP weight for 0.10 window size

### PR DESCRIPTION
**Pull request by BoltzmannBrain**
FP weight scaled down; issue #95

### CODE REVIEW

**Comment by BoltzmannBrain** [comment]
@subutai please review. Closes issues #95 #98 #77 #36

File: nab/labeler.py [hunk 1]
```diff
@@ -349,6 +357,20 @@ def checkWindows(self):
...
             del windows[i+1]
 
+      length = len(self.corpus.dataFiles[relativePath].data)
+      probationIndex = int(math.ceil(self.probationaryPeriod*length))
+      probationTimestamp = self.corpus.dataFiles \
+                             [relativePath].data["timestamp"][probationIndex]
+      if num_windows > 0:
+        if (pandas.to_datetime(windows[0][0])
+            -probationTimestamp).total_seconds() < 0:
+#          raise ValueError("The first window in %s overlaps with the"
```
Comment (subutai) [inline nab/labeler.py]: In general, shouldn't we raise an exception if this happens?

**Comment by subutai** [comment]
Left one question. Other than that :+1:

File: nab/labeler.py (hunk 1 above)
Comment (BoltzmannBrain) [inline nab/labeler.py]: Yes, but we want the code to either delete the bad window, or remove the file completely, and then keep running. Are you saying the ValueError should be raised in a try-except block?

Comment (subutai) [inline nab/labeler.py]: No, I mean an exception. In general I don't think we should proceed without manual intervention in the case where the probationary period overlaps the first anomaly window.

**Comment by BoltzmannBrain** [comment]
Okay, so the ValueError exception is raised for the data file "realAWSCloudwatch/iio_us-east-1_i-a2eb1cd9_NetworkIn.csv". Should we remove it from the benchmark dataset? Removing the window would result in FPs.

**Comment by BoltzmannBrain** [comment]
@subutai please review. As we add more data files and other instances of poor labels arise, we can remove them in the same function -- `removePoorLabels()`.

**Pull request merged**